### PR TITLE
Save Contract Type to Local storage

### LINKED ISSUE #156: Local storage taking wrong commit
Salt saved in local storage does not reflect what registrar was used to submit it.

### PR DESCRIPTION
**Pull request by jessgusclark**
Changes local storage to hold a json object with hash and contract type. 

Closes #156.

### CODE REVIEW

File: src/app/tabs/registrar/containers/RevealContainer.js [hunk 1]
```diff
@@ -20,14 +20,14 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   // eslint-disable-next-line max-len
```
Comment (ilanolkies) [inline src/app/tabs/registrar/containers/RevealContainer.js]: Can we remove this two eslint disables? We should follow linting :)

File: src/app/tabs/registrar/operations.js [hunk 2]
```diff
@@ -87,17 +87,29 @@ export const commit = (domain, setupAddr) => async (dispatch) => {
             return resolve(dispatch(notifyError(_error.message)));
           }
 
-          localStorage.setItem(`${domain}-salt`, salt);
+          localStorage.setItem(`${domain}-options`, JSON.stringify({
+            salt,
+            contract: setupAddr ? 'FIFSADDR' : 'FIFIS',
```
Comment (ilanolkies) [inline src/app/tabs/registrar/operations.js]: Move this constants to `constants.js`

File: src/app/tabs/registrar/containers/RevealContainer.js (hunk 1 above)
Comment (jessgusclark) [reply src/app/tabs/registrar/containers/RevealContainer.js]: fixed.

File: src/app/tabs/registrar/operations.js (hunk 2 above)
Comment (jessgusclark) [reply src/app/tabs/registrar/operations.js]: fixed

**Pull request merged**